packages/sample-app/src/app-minimal.tsx

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="./types.d.ts" />
+
+import '@patternfly/react-core/dist/styles/base.css';
+import './app-minimal.css';
+
+import * as React from 'react';
+import { render } from 'react-dom';
+import MinimalAppInit from './components/app-minimal/MinimalAppInit';
+import MinimalAppPage from './components/app-minimal/MinimalAppPage';
+import ErrorBoundary from './components/common/ErrorBoundary';
+import Loading from './components/common/Loading';
+import PageHeader from './components/common/PageHeader';
+import PageLayout from './components/common/PageLayout';
```

Comment:
@karelhala Using code splitting with webpack `import()` is something I tried to avoid in the sample host application.

The goal is to keep code simple, this is the kind of change that you'd normally do in real world applications when you want to defer loading a component to a time when it's really needed.

In this case, all of these components are needed to render a single page component (`MinimalAppPage`) so I don't see much benefit vs. complexity that we would gain here.

Reply:
Understandable I was more worried about exporting `React.lazy` instead of regular components.